spec/models/cart_spec.rb

```diff
@@ -9,21 +9,21 @@
     it "successfully adds a beer and quantity to the cart" do 
       cart = Cart.new({})
       cart.add_beer(beer, 1)
-      expect(cart.contents).to eq({beer.id.to_s=>{"beer_name"=>"Yeti", "beer_price"=>606, "quantity"=>1}})
+      expect(cart.contents).to eq({beer.id.to_s=>{"quantity"=>1}})
```

Comment:
Redundant curly braces around a hash parameter.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.